From a128c361d828cfc1fc1a5468d61f152b4900d539 Mon Sep 17 00:00:00 2001
Message-Id: <a128c361d828cfc1fc1a5468d61f152b4900d539.1499608047.git.sleepingzucchini@gmail.com>
In-Reply-To: <cover.1499608047.git.sleepingzucchini@gmail.com>
References: <cover.1499608047.git.sleepingzucchini@gmail.com>
From: Tyler Olivieri <sleepingzucchini@gmail.com>
Date: Sat, 8 Jul 2017 20:07:50 -0400
Subject: [PATCH 1/5] staging: ccree: remove redudant semicolons

Patch to remove checkpatch warning:
WARNING: Statements terminations use 1 semicolon

Signed-off-by: Tyler Olivieri <sleepingzucchini@gmail.com>
---
 drivers/staging/ccree/ssi_buffer_mgr.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/staging/ccree/ssi_buffer_mgr.c b/drivers/staging/ccree/ssi_buffer_mgr.c
index b35871e..c05cd67 100644
--- a/drivers/staging/ccree/ssi_buffer_mgr.c
+++ b/drivers/staging/ccree/ssi_buffer_mgr.c
@@ -150,7 +150,7 @@ static inline int ssi_buffer_mgr_render_buff_to_mlli(
 	u32 **mlli_entry_pp)
 {
 	u32 *mlli_entry_p = *mlli_entry_pp;
-	u32 new_nents;;
+	u32 new_nents;
 
 	/* Verify there is no memory overflow*/
 	new_nents = (*curr_nents + buff_size / CC_MAX_MLLI_ENTRY_SIZE + 1);
-- 
2.9.4

